### PR TITLE
Fixes #445

### LINKED ISSUE #445: Wrong group in init script
### Is this a bug or a feature request?
Bug

### What did you expect?
The init script will use the correct user and group for the new network-server

### What happened?
The init script used the wrong group (chirpstack-network-server) to start the chirpstack-network-server. This resulted in errors during the package installation.

### What version are your using?
v3.4.0

### How can your issue be reproduced?
Install the latest chirpstack-network-server debian package

### Could you share your log output?
```Reading package lists... Done
Building dependency tree       
Reading state information... Done
0 upgraded, 0 newly installed, 0 to remove and 10 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up loraserver (3.4.0) ...
ln: failed to create symbolic link ‘/etc/init.d/chirpstack-network-server’: File exists
 * Starting ChirpStack Network Server                                                                                                                                                                                                         start-stop-daemon: group 'chirpstack-network-server' not found
                                                                                                                                                                                                                                       [fail]
dpkg: error processing package loraserver (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 loraserver
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

### PR DESCRIPTION
**Pull request by FreeK**
Fixes issue #445

### CODE REVIEW

File: packaging/files/chirpstack-network-server.init [hunk 1]
```diff
@@ -11,8 +11,8 @@
 
 NAME=chirpstack-network-server
 DESC="ChirpStack Network Server"
-DAEMON_USER=chirpstack-network-server
-DAEMON_GROUP=chirpstack-network-server
+DAEMON_USER=networkserver
```
Comment (brocaar) [inline packaging/files/chirpstack-network-server.init]: Good catch :+1:

File: packaging/scripts/post-install.sh [hunk 2]
```diff
@@ -32,12 +32,22 @@ function restart_service {
 	fi	
 }
 
+function create_logdir {
+	if [[ ! -d $LOG_DIR ]]; then
+		mkdir -p $LOG_DIR
+		chown -R $DAEMON_USER.$DAEMON_GROUP $LOG_DIR
```
Comment (brocaar) [inline packaging/scripts/post-install.sh]: I believe this should be `chown -R $DAEMON_USER:$DAEMON_GROUP $LOG_DIR` (thus `.` replaced by `:`)?

> chown [OPTION]... [OWNER][:[GROUP]] FILE...

Comment (FreeK) [reply packaging/scripts/post-install.sh]: Should work with `.` and `:`
https://serverfault.com/a/194299

But to be on safe `:` is a good idea.

Comment (FreeK) [reply packaging/scripts/post-install.sh]: I've updated my master for both, chirpstack-network-server and chirpstack-application-server

**Pull request merged**